Sources/Sentry/Public/SentrySpan.h

```diff
@@ -1,9 +1,93 @@
-#import <Foundation/Foundation.h>
+#import "SentrySpanContext.h"
+#import <Sentry/Sentry.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+@class SentryTransaction, SentrySpanId, SentryId, SentryHub;
+
 NS_SWIFT_NAME(Span)
-@protocol SentrySpan <NSObject>
+@interface SentrySpan : SentrySpanContext
+SENTRY_NO_INIT
+
+/**
+ * The timestamp of which the span ended.
+ */
+@property (nullable, nonatomic, strong) NSDate *timestamp;
+
+/**
+ * The start time of the span.
+ */
+@property (nullable, nonatomic, strong) NSDate *startTimestamp;
+
+/**
+ * An arbitrary mapping of additional metadata of the span.
+ */
+@property (nullable, readonly) NSDictionary<NSString *, id> *extras;
+
+/**
+ * Whether the span is finished.
+ */
+@property (readonly) bool isFinished;
```

Comment:
`h`: Shouldn't this be BOOL?

Reply:
To be fair it will work the same.
I do prefer the c version bool.  BOOL is a char depending on the platform so it can have multiples values. 
But since Obj-C frameworks uses BOOL let's stick to it.

https://developer.apple.com/documentation/objectivec/bool